publishers/community/slack/slack_layout.py

```diff
@@ -125,8 +125,8 @@ def publish(self, alert, publication):
         publication['@slack.attachments'].append({
             'color': self._color(),
             'fields': [
-                {'title': key.capitalize(), 'value': rule_presentation['fields'][key]}
-                for key in rule_presentation['fields'].keys()
+                {'title': key.capitalize(), 'value': value}
+                for key, value in rule_presentation['fields'].items()
```

Comment:
this is python2 so please use `.iteritems()` here. in python2, `.items()` creates copies of the key/values as tuples in memory and is more costly.

Reply:
Got it. Pushing a fix!